### PR TITLE
Support Email Address Fixed in the Readme.md

### PR DESCRIPTION
**Pull request by seharbat00l**
Issue #1716 : resolved

### CODE REVIEW

**Review by misl6 — COMMENTED** [review]
Hi @seharbat00l !

I've left a review, let me know what are your thoughts about it 😀

(We will need to fix the same issue over all the Kivy's Github repos 😅)

File: README.md [hunk 1]
```diff
@@ -168,8 +168,7 @@ config, along with the environment variables that would override them.
 
 If you need assistance, you can ask for help on our mailing list:
```
[the remark this answers is not in the document]
Comment (seharbat00l) [reply README.md:169]: Yes we can do that. You want to have the same clickable text like "Kivy Users Google Group" for Discord? Am i right?

[the remark this answers is not in the document]
Comment (seharbat00l) [reply README.md:169]: But is it required only on this repo or others too? If yes then which ones? You can create issue for them and I'll refactor them.

[the remark this answers is not in the document]
Comment (seharbat00l) [reply README.md:169]: I'll make another PR with this Discord refactor on this repo. Kindly approve that PR.

**Comment by seharbat00l** [comment]
I've made the commit after refactoring support section. You can checkk

**Review by misl6 — APPROVED** [review]
LGTM. Thank you!

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (misl6) [reply README.md]: > But is it required only on this repo or others too? If yes then which ones? You can create issue for them and I'll refactor them.

Yes, it's needed also on other repos of the `kivy` organization. I will open the issues and assign them to you. 😀

[the remark this answers is not in the document]
Comment (misl6) [reply README.md]: > But is it required only on this repo or others too? If yes then which ones? You can create issue for them and I'll refactor them.

Ping me on the following issues, so I can assign it to you:
- https://github.com/kivy/kivy/issues/8409
- https://github.com/kivy/python-for-android/issues/2899
- https://github.com/kivy/pyjnius/issues/686
- https://github.com/kivy/plyer/issues/776
- https://github.com/kivy/kivy-ios/issues/872

**Pull request merged**